src/app/components/layout/navbar/navbar.component.ts

```diff
@@ -155,10 +156,17 @@ export class NavbarComponent implements OnInit, AfterViewInit, OnDestroy {
           northboundServices = sortBy(northboundServices, function (obj) {
             return obj.name.toLowerCase();
           });
-          const notificationService = servicesData.filter((el => (el.type === 'Notification')));
-          const managementService = servicesData.filter((el => (el.type === 'Management')));
-          const dispatcherService = servicesData.filter((el => (el.type === 'Dispatcher')));
-          const bucketStorageService = servicesData.filter((el => (el.type === 'BucketStorage')));
+
+          // FYI; Four additional services exist in the Fledge and two in FogLAMP. There will be some extra API calls in Fledge (to search all of the expected additional services in different API response) because we don't have any info whether user is FogLAMP or Fledge.
+          let expectedExternalServiceType = ['Notification', 'Management', 'Dispatcher', 'BucketStorage'];
+
+          const additionalServices = servicesData.filter((s) => expectedExternalServiceType.includes(s.type))
+
+          // If service is failed or unresponsive, check schedule as well
+          const failedOrUnresponsiveServices = additionalServices.filter((s) => ['failed', 'unresponsive'].includes(s.status.toLowerCase()));
+          if (failedOrUnresponsiveServices.length > 0) {
+            this.getSchedules();
```

Comment:
For navbar thing, why do you need schedules data. i.e. simply a response of /service endpoint (from service registry)

Reply:
This is to get the schedule status on the Service configuration modal if service is unresponsive or failed.